### ISSUE TITLE
Command C-u C-c C-c does not work

### ISSUE THREAD

**Issue by DungxNguyen** [jorgenschaefer/elpy#1225]
Hi, I can not make C-u C-c C-c to work. I tried to debug the problem but I can not find the function
`elpy-shell-send-region-or-buffer` in elpy-shell.el, though I know it is bound to C-c C-c.
Otherwise, I find the function 
`elpy-shell--send-region-or-buffer-internal`
is actually called each time I press C-u C-c C-c from the function
`elpy-shell-send-region-or-buffer-and-step`

And I can see that the function `elpy-shell-send-region-or-buffer-and-step` does not accept any prefix argument, so the argument can not be passed to the shell, which means the argument `arg` is always nil even I press C-u prior to the C-c C-c.

My emacs version is 25.3. Elpy version 20171124.720 from melpa.

**Comment by DungxNguyen** [comment]
Can somebody tell me where the function `elpy-shell-send-region-or-buffer` is?

**Comment by rgemulla** [comment]
The function ```elpy-shell-send-region-or-buffer``` is auto-generated by line

```elisp
(elpy-shell--defun-step-go elpy-shell-send-region-or-buffer-and-step)
```

in ```elpy-shell.el```. The prefix argument is currently not passed along; I'll push a fix as soon as I can find some time. (It would be helpful if you could provide a short example to test it.)

Until the fix is in, you can use ```elpy-shell--send-region-or-buffer-internal```, which does accept the prefix argument, but does not support the input/output echoing features of elpy-shell (which is what is added by ```elpy-shell-send-region-or-buffer```).

**Comment by rgemulla** [comment]
I created a PR that fixes this issue (#1226).

**Comment by rgemulla** [comment]
Pushed the PR. This should fix this issue.
